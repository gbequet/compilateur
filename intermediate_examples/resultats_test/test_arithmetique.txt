2
2
2
2
0000
8
8
8
8
9
3
5
3
210
10
9
9
9
9
