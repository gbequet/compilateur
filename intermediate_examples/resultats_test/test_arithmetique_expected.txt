2
2
2
2
-1
0
0
0
8
8
8
8
9
3
5
3
210
10
9
9
9
9
